### ISSUE TITLE
panic when updating 0.14.1 => 0.14.2

### ISSUE THREAD

**Issue by SFort** [greshake/i3status-rust#904]
```
i3status-rs i3status-rs.toml
```
```
{"version": 1, "click_events": true}
thread 'main' panicked at 'Could not find disk drive icon', src/blocks/disk_space.rs:215:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
<details>
<summary>
i3status-rs.toml
</summary>

```

theme = "modern"

[[block]]
block = "github"
interval = 300
format = "{total}G"

[icons.overrides]
cpu = "¦ "
net_up = " ^"
net_down = " v"

[[block]]
block = "disk_space"
path = "/"
alias = ""
interval = 40

[[block]]
block = "memory"
display_type = "memory"
icons = false
clickable = false
format_mem = "{Mup}%"
format_swap = "{Mup}%"

[[block]]
block = "cpu"
interval = 2

[[block]]
block = "net"
device = "eno1"

#[[block]]
#block = "sound"
#step_width = 2
#on_click ="amixer set Master toggle > /dev/null"

[[block]]
block = "time"
interval = 30
format = "%a %d|%m %T"

```
</details>

**Comment by ammgws** [comment]
Also paste your theme file ("modern")

**Comment by ammgws** [comment]
Was able to reproduce. To fix it add the following to your config:

```
[icons]
name = "none"
```

Will need to look into this a bit more later.

**Comment by SFort** [comment]
using that fix creates unwanted results ("GITHUB", "ETH"...) making the bar too wide
![Original](https://user-images.githubusercontent.com/1879846/98341886-7b465980-2007-11eb-81b2-a18b3e5a1243.png)
![With fix](https://user-images.githubusercontent.com/1879846/98341906-84372b00-2007-11eb-8b58-06a8430b45ef.png)

**Comment by ammgws** [comment]
Bisects to 8074b0bbade191c0ab31158e666efed0fa392f04 (https://github.com/greshake/i3status-rust/pull/714 Add icon support via format string to Disk Space block)

Added a fix in #904 (assuming no other blocks suffer from the same issue)
